Jenkinsfile

```diff
@@ -9,32 +9,42 @@ stage('Initialize') {
 
 provision(devEnvironmentName())
 if (isMaster()) {
-    input 'Deploy to Production'
     provision("prod")
 }
 
 def provision(environment) {
-    stage('Provision Infrastructure Stack') {
+    def label = ((environment.startsWith("dev")) ? "dev" : environment).
+        toUpperCase();
+
+    if (label != "DEV") {
+        stage("master") {
+            timeout(time:5, unit:'DAYS') {
```

Comment:
Just curious the "timeout" command - I think this is a newer one they added to the syntax? So it kills the pipeline run if no action is taken by the end of the timeout? And you chose 5 days as sort of an arbitrary number?